lightning/src/ln/channel.rs

```diff
@@ -4093,9 +4093,6 @@ pub(super) struct DualFundingChannelContext {
 // Counterparty designates channel data owned by the another channel participant entity.
 pub(super) struct Channel<SP: Deref> where SP::Target: SignerProvider {
 	pub context: ChannelContext<SP>,
-	pub dual_funding_channel_context: Option<DualFundingChannelContext>,
```

Comment:
Looks like this was added prior to this PR. Let's remove it in an independent commit rather than a fixup so that it is clear why it was removed.